actions/store_command_info_MOD.js

```diff
@@ -117,7 +118,7 @@ module.exports = {
         ? jp.query(this.getDBM().Files.data.commands, `$..[?(@._id=="${this.evalMessage(data.valueToSearch, cache)}")]`)
         : jp.query(
             this.getDBM().Files.data.commands,
-            `$..[?(@.name=="${cache.msg.content
+            `$..[?(@.name=="${(interaction ?? msg).content
               .slice(this.getDBM().Files.data.settings.tag.length || cache.server.tag.length)
               .split(/ +/)
               .shift()}")]`,
```

Comment:
This should be `interaction?.commandName ?? msg...`. But let's extract it to a variable.